### PR TITLE
Folderize Docs: Each topic gets a folder

### PR DESCRIPTION
**Pull request by herostrat**
Mirrors the structure on the webpage. I find it easier to find things.

### CODE REVIEW

**Comment by blakeblackshear** [comment]
Can you see the netlify logs? The build fails because of broken links.

**Comment by herostrat** [comment]
> Can you see the netlify logs? The build fails because of broken links.
Yes, I should really mark things WIP in the future. I ran out on time to update all links.

**Comment by herostrat** [comment]
Something seems to be broken with netlify. I cannot get the pipeline to run, there are no logs.
I think I fixed all broken links nonetheless.

**Comment by blakeblackshear** [comment]
```
2:02:42 PM: Exhaustive list of all broken links found:
2:02:42 PM: 
2:02:42 PM: - On source page path = /configuration/rtmp:
2:02:42 PM:    -> linking to /faqs#audio-in-recordings (resolved as: /faqs)
2:02:42 PM: 
2:02:42 PM: - On source page path = /guides/getting_started:
2:02:42 PM:    -> linking to /installation
2:02:42 PM: 
2:02:42 PM: - On source page path = /troubleshooting/faqs:
2:02:42 PM:    -> linking to /installation#calculating-required-shm-size (resolved as: /installation)

**Comment by herostrat** [comment]
My search regex was incomplete. Thanks for the log.
I thought one has to add the .md to the links.

I cannot few the logs of my new commits, where I tried to sync all links.

File: docs/docs/integrations/home-assistant.md [hunk 1]
```diff
@@ -134,7 +134,7 @@ For advanced usecases, this behavior can be changed with the [RTMP URL
 template](#options) option. When set, this string will override the default stream
 address that is derived from the default behavior described above. This option supports
 [jinja2 templates](https://jinja.palletsprojects.com/) and has the `camera` dict
-variables from [Frigate API](https://blakeblackshear.github.io/frigate/usage/api#apiconfig)
```
Comment (herostrat) [inline docs/docs/integrations/home-assistant.md:137]: This link was outdated (1/2)

File: docs/docs/integrations/home-assistant.md [hunk 2]
```diff
@@ -171,7 +171,7 @@ The Frigate integration seamlessly supports the use of multiple Frigate servers.
 In order for multiple Frigate instances to function correctly, the
 `topic_prefix` and `client_id` parameters must be set differently per server.
 See [MQTT
-configuration](https://blakeblackshear.github.io/frigate/configuration/index#mqtt)
```
Comment (herostrat) [inline docs/docs/integrations/home-assistant.md:174]: This link was outdated (2/2)

**Comment by blakeblackshear** [comment]
`npm run build` should show you the same output

**Pull request merged**